Bottom: d714d0130a01ee14eac519e6d349f09cb04a581a
Top:    4a94f0a0a0345f134da23d76e34447998bd3437c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-27 17:15:13 +0000

Refresh of hid-gtk-compute-warp-pointer-o

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index a0804cb..6b2dd87 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -82,14 +82,6 @@ ghid_pan_fixup ()
       return;
     }
 
-  /* if we can see the entire board and some, then zoom to fit */
-  if (gport->view_width > PCB->MaxWidth &&
-      gport->view_height > PCB->MaxHeight)
-    {
-      zoom_by (1, 0, 0);
-      return;
-    }
-
   ghidgui->adjustment_changed_holdoff = TRUE;
   gtk_range_set_value (GTK_RANGE (ghidgui->h_range), gport->view_x0);
   gtk_range_set_value (GTK_RANGE (ghidgui->v_range), gport->view_y0);
